README.md

```diff
@@ -29,6 +32,21 @@ cd omero-guide-python
 repo2docker .
 ```
 
+Building locally using conda and Jupyter:
+
+ * Create a conda environment i.e. ``conda env update -n stardist -f binder/environment.yml``
+ * Activate the newly created environment
+
+The following steps are only required if you want to run the notebooks
+ * If you have Anaconda installed:
```

Comment:
This does not seem logical cf. the paragraph below.

Reply:
Anaconda is correct